packages/wallet/src/redux/protocols/indirect-defunding/reducer.ts

```diff
@@ -44,14 +46,22 @@ export const initialize = (
       };
     }
     newSharedData = signResult.store;
+    // send a request for opponent to start new defunding process first, because they may not yet have done so
+
+    const actionToRelay = defundRequested({
+      processId: '0XTODO', // wish to terminate existing process in opponent's wallet (could use currentProcessId but this is not on SharedData)
```

Comment:
Do we expect there to be an opponent's process that would be terminated here or would that already be handled by the challenging reducers?

Reply:
The opponent might still be running that process: for example, they are at the final stage of `Dispute`, and haven't yet decided whether to defund or not.

Reply:
Have moved `currentProcessId?` to sharedData and substituted it for '0xTODO` here (https://github.com/magmo/apps/pull/530/commits/4e1fc8a2317d5098fed7a54cc0efc5b71455fb6a)